pypackify.py

```
@@ -9,8 +11,23 @@ def load_config(config_path="config.json"):
+        raise FileNotFoundError(f"Licenses file '{licenses_path}' not found.")
+    with open(licenses_path, "r") as file:
+        licenses = json.load(file)
+    return licenses.get(license_name, f"License text for '{license_name}' not found.")
+
+def validate_version(version):
+    """Validate that the version string follows semantic versioning."""
+    semver_pattern = r"^\d+\.\d+\.\d+$"
+    if not re.match(semver_pattern, version):
+        raise ValueError(f"Invalid version format: '{version}'. Expected format: X.Y.Z (e.g., 1.0.0)")
+
 def create_package(config_path="config.json"):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_package`

Add return type annotation: `None`

(ANN201)